pixl_dcmd/src/pixl_dcmd/__init__.py

```diff
@@ -16,18 +16,12 @@
 import logging
 
 from pixl_dcmd.main import (
-    apply_tag_scheme,
-    enforce_whitelist,
-    remove_overlays,
+    anonymise_dicom,
     write_dataset_to_bytes,
+    should_exclude_series,
 )
 
-__all__ = [
-    "remove_overlays",
-    "write_dataset_to_bytes",
-    "apply_tag_scheme",
-    "enforce_whitelist",
-]
+__all__ = ["anonymise_dicom", "write_dataset_to_bytes", "should_exclude_series"]
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
```

Comment:
ah logging, `pixl_dcmd` is a library so shouldn't really be setting the logging level, commented later down the line but we should set the logging level in the `pixl.py` of orthanc raw, and replace the logging within pixl_dcmd to use `loguru` 
this should be the only interaction with the logger (other than logging) that we do in `dcmd`
```
from loguru import logger
```